packages/components-native/package.json

```diff
@@ -2,6 +2,16 @@
   "name": "@jobber/components-native",
   "version": "0.28.1",
   "license": "MIT",
+  "description": "React Native implementation of Atlantis",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/GetJobber/atlantis.git"
+  },
+  "author": "The Frends",
+  "bugs": {
+    "url": "https://github.com/GetJobber/atlantis/issues"
+  },
+  "homepage": "https://github.com/GetJobber/atlantis#readme",
```

Comment:
Nice! Thanks for adding it

Reply:
No worries, it was needed for the podspec file